### ISSUE TITLE
Use a private network range for Pod CIDR

### ISSUE THREAD

**Issue by hoeghh** [hoeghh/kubernetes_the_easy_way#29]
The network range 20.0.0.0/16 is in the public network area (this is owned by Microsoft)

https://en.wikipedia.org/wiki/Private_network

In time, this should be configurable in the ```config``` file

**Comment by hoeghh** [comment]
Maybe use the range 172.16.0.0/12 .
Test, and if it works, replace the current address space.

**Comment by hoeghh** [comment]
Files to be changed : 
```
grep -R "20.0.0.0/16" * | cut -d ":" -f1
scripts/install-master.sh
scripts/post_deploy.sh
scripts/install-worker.sh
scripts/install-loadbalancer.sh
```

**Comment by hoeghh** [comment]
Working on this. Needs testing before merge.

**Comment by hoeghh** [comment]
Fixed with 14af8ef80670805945429f6b34705137306ba3e7
